AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -374,120 +378,115 @@ - (void)testLongListOfSubChildren {
   }
 }
 
-- (void)testSetAndRemoveEventProperty {
```

Comment:
How is this case being covered now?  Why was this test removed?

Reply:
It's been divided into smaller tests for more readability :)